chartfx-math/src/test/java/de/gsi/math/FastMathTests.java

```diff
@@ -22,87 +22,87 @@
 
     @Test
     public void cosineIdentityChecks() {
-        double precision = FastMath.getPrecision();
-        int range = FastMath.getPrecision();
+        double precision = MathBaseFast.getPrecision();
+        int range = MathBaseFast.getPrecision();
         for (int i = -range; i < range; i++) {
-            final float x = (float) (2.0 * Math.PI * i / precision);
-            final float val1 = (float) Math.cos(x);
-            final float val2 = FastMath.cos(x);
-            final float diffApproximation = Math.abs(val1 - val2);
+            final float x = (float) (2.0 * java.lang.Math.PI * i / precision);
+            final float val1 = (float) java.lang.Math.cos(x);
+            final float val2 = MathBaseFast.cos(x);
+            final float diffApproximation = java.lang.Math.abs(val1 - val2);
             assertTrue(diffApproximation <= 2e-3, i + ": cos(" + x + ") diff (float) = " + diffApproximation);
         }
 
         for (int i = -range; i < range; i++) {
-            final double x = 2.0 * Math.PI * i / precision;
-            final double val1 = Math.cos(x);
-            final double val2 = FastMath.cos(x);
-            final double diffApproximation = Math.abs(val1 - val2);
+            final double x = 2.0 * java.lang.Math.PI * i / precision;
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.PI' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)